src/commands/InitCommand.ts

```diff
@@ -33,6 +34,10 @@ export class InitCommand implements yargs.CommandModule {
             })
             .option("docker", {
                 describe: "Set to true if docker-compose must be generated as well. False by default."
+            })
+            .option("pm", {
```

Comment:
Can we instead just make a flag for `yarn` and install with `npm` by default?  What does webpack do?

Reply:
I thought of making npm default option if flag is not provided. We can make one flag for yarn installation only. Need your confirmation here.

Reply:
@imnotjames, need your final confirmation here.

Reply:
default should be npm, if users use `init` command without `pm` option specified - it should work and default should be npm.

Reply:
done